plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -12,6 +12,18 @@ import ServiceConfigUtil from '../utils/ServiceConfigUtil';
 import ServiceConfigBaseSectionDisplay from './ServiceConfigBaseSectionDisplay';
 import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
 
+const getNetworkType = (networkType, appDefinition) => {
```

Comment:
I was about to ask why not using `function` here and decided to research a bit...
Here what I've found and I don't know how to live my life anymore

http://stackoverflow.com/a/28135120

![tea](https://cloud.githubusercontent.com/assets/186223/21846122/df2e9808-d7f5-11e6-8a88-05cdf70e6ed9.gif)


Do we really want a top level arrow functions bound to a `const`?
// cc @rcorral @orlandohohmeier